ParseNOTAM.py

```diff
@@ -10,15 +10,14 @@ def ParseNOTAM(json_data = None):
 
         # Reading JSON data from the file
         with open(file_path, 'r') as json_file:
-            json_data = json.load(json_file)
+            json_data_test = json.load(json_file)

```

Comment:
seems odd you are changing the variable name here from `json_data` to `json_data_test` when the code below still refers to `json_data`